finesse/gui/stepper_motor_view.py

```diff
@@ -53,5 +65,21 @@ def _preset_clicked(self, btn: QPushButton) -> None:
         # If the motor is already moving, stop it now
         pub.sendMessage(f"device.{STEPPER_MOTOR_TOPIC}.stop")
 
+        pub.sendMessage(f"device.{STEPPER_MOTOR_TOPIC}.notify_on_stopped")
         target = float(self.angle.value()) if btn is self.goto else btn.text().lower()
         pub.sendMessage(f"device.{STEPPER_MOTOR_TOPIC}.move.begin", target=target)
+
+    def _update_mirror_position_display(self, moved_to: float | None) -> None:
+        """Display the angle the mirror has moved to.
+
+        moved_to will be None if mirror is moving, in which case display "Moving...",
+        otherwise, display angle or its associated name.
+        """
+        if moved_to is None:
+            self.mirror_position_display.setText("Moving...")
+        else:
+            preset = next((k for k, v in ANGLE_PRESETS.items() if v == moved_to), None)
+            if preset:
+                self.mirror_position_display.setText(preset.upper())
+            else:
+                self.mirror_position_display.setText(f"{moved_to}°")
```

Comment:
On reflection, I'm thinking it might be useful to see the angle for presets too. How about:

```suggestion
            text = f"{moved_to}°"
            if preset := next((k for k, v in ANGLE_PRESETS.items() if v == moved_to), None):
                text += f" ({preset})"
            self.mirror_position_display.setText(text)
```

Reply:
This does display the names as, e.g. "cold_bb", rather than the text on the labels ("COLD BB"), but it's still obvious which each corresponds to haha